components/cv_form/projects.tsx

```diff
@@ -2,6 +2,7 @@ import { Field, FieldArray } from "formik";
 import * as Icons from "../Icons";
 import * as Options from "../../constants/CvFormOptions";
 import NestedRow from "./nested_row";
+import DatePicker from "../datepicker";
```

Comment:
Let's import component as named - `MonthYearDatePicker` to avoid confusion about what component is used.